setup.py

```diff
@@ -17,18 +17,18 @@ def has_ext_modules(self):
 
 setuptools.setup(
     name="tf-big",
-    version="0.1.3",
+    version="0.2.0",
     packages=setuptools.find_packages(),
     package_data={
         '': ['*.so'],
     },
     python_requires=">=3.5",
     install_requires=[
         "numpy >=1.14",
-        "tensorflow >=1.13.1, <2",
+        "tensorflow ==2.1.0",
```

Comment:
any reason we're not shooting for > 2.0.0?

Reply:
The library has to be compiled for specific TF versions, so if anything I'd rather add a specific list of versions we support. This should be straight forward to do, just a question of updating the Circle CI configuration, but wanted to use Circle CI matrix in that case instead of the current approach.

I've created https://github.com/tf-encrypted/tf-big/issues/62